<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StakeForCharityDialog</class>
 <widget class="QWidget" name="StakeForCharityDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stake For Charity</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>650</width>
        <height>298</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>Percentage of stake to send</string>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&gt;Percentage:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_3">
           <property name="toolTip">
            <string>Min amount to send</string>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&gt;Min Amount:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_4">
           <property name="toolTip">
            <string>Max amount to send</string>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&quot;&gt;Max Amount:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLineEdit" name="charityPercentEdit">
           <property name="toolTip">
            <string>Enter whole numbers 1 - 50</string>
           </property>
           <property name="placeholderText">
            <string>Enter % to Give (1-50)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" colspan="2">
          <widget class="QLineEdit" name="charityMinEdit">
           <property name="toolTip">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Enter Min Amount (optional)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5" colspan="2">
          <widget class="QLineEdit" name="charityMaxEdit">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Enter Max Amount (optional)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="toolTip">
            <string>Address to send portion of stake to</string>
           </property>
           <property name="text">
            <string>Recieving Address:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QLineEdit" name="charityAddressEdit">
           <property name="toolTip">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Enter Address to Send to</string>
           </property>
          </widget>
         </item>
         <item row="3" column="6">
          <widget class="QPushButton" name="addressBookButton">
           <property name="toolTip">
            <string>Choose an address from the address book</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../bitcoin.qrc">
             <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+A</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_6">
           <property name="toolTip">
            <string>The amount left over returned to you</string>
           </property>
           <property name="text">
            <string>Change Address:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QLineEdit" name="charityChangeAddressEdit">
           <property name="toolTip">
            <string>Change address must be owned by you</string>
           </property>
           <property name="placeholderText">
            <string>Enter Change Address (optional)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="6">
          <widget class="QPushButton" name="changeAddressBookButton">
           <property name="toolTip">
            <string>Choose an address from the address book</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../bitcoin.qrc">
             <normaloff>:/icons/address-book</normaloff>:/icons/address-book</iconset>
           </property>
           <property name="shortcut">
            <string>Alt+A</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="enableButton">
           <property name="toolTip">
            <string/>
           </property>
           <property name="statusTip">
            <string>Check and turn on Stake For Charity</string>
           </property>
           <property name="text">
            <string>Enable</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="disableButton">
           <property name="statusTip">
            <string>Turn off Stake For Charity</string>
           </property>
           <property name="text">
            <string>Disable</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="7">
          <widget class="QLabel" name="label_2">
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span&gt;Stake For Charity allows you to automatically send up to 50% of your stake to another Netcoin address after it matures.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span&gt;Please enter percentage to give and Netcoin address to give to. Min and Max amount and change address are optional.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="4" colspan="3">
          <widget class="QLabel" name="message">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>170</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>170</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>170</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>170</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>120</red>
                 <green>120</green>
                 <blue>120</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>120</red>
                 <green>120</green>
                 <blue>120</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="0">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
